eblob_kit.py

```diff
@@ -630,13 +785,12 @@ def check_header(self, header):
 
     def check_hole(self, position, end):
         """Check headers in data in area [@position, @end)."""
-        logging.error('I have found hole in data %s which is not '
-                      'covered by valid headers from index',
+        logging.error('I have found hole in data %s which is notcovered by valid headers from index',
```

Comment:
Why was it changed?)

Reply:
If I only could remember, ) would change it back.